### DISCUSSION TITLE
Replace `Deno.run` with `Deno.Command`

### DISCUSSION THREAD

**Discussion by Leokuma** [Hexagon/pup#11] [Ideas]
Firstly, I'm really excited about this project and looking forward to 1.0 🎉.

https://github.com/Hexagon/pup/blob/9e634f98ed5bbaab65edb81bb6f298dc6b76379f/lib/core/runner.ts#L66

[`Deno.run` is intended to be deprecated in favor of `Deno.Command`](https://deno.com/blog/v1.31#api-stabilizations). Also, `Deno.Command` is stabilized, whereas `Deno.run` still requires `--unstable`.

**Comment by Hexagon** [comment]
👋 

Interesting! Will have a look at moving to Deno.Command 👍

**Reply by Hexagon** [reply]
@Leokuma Done in latest commit, i'll release a new version after some testing.

Check out the changes of lib/core/runner.ts if you want to dig into the details 😄 

https://github.com/Hexagon/pup/commit/bb2e9638942320c849abf1ed5b65b085a16445f0#diff-0a0968976a8fee548c86fcaf957c79808eda1609021a45d5ec0b56a409f97ae0

**Reply by Leokuma** [reply]
You're the best. You're adding so many features. V1 will come out with plenty of features already 💪🎉.

When the code gets more stabilized - maybe after v1 -, I might try to contribute with something.
